### PR TITLE
Add debian/ dir for deb packaging

### PR DESCRIPTION
**Pull request by hosiet**
Unfortunately current CPack codes are broken in newer debian-based systems. For example, `libwebsockets8` no longer exists. Thus some canonical packaging instructions might help people with installing ttyd.

As per request debiancn/repo#74, I wrote packaging instructions for ttyd here and provide them as-is.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ cmake,
+ pkg-config,
+ libwebsockets-dev,
+ libjson-c-dev,
+ libssl-dev,
+ xxd,
+Standards-Version: 3.9.8
+Homepage: https://tsl0922.github.io/ttyd/
+Vcs-Git: https://github.com/hosiet/ttyd.git
+Vcs-Browser: https://github.com/hosiet/ttyd.git
```
Comment (tsl0922) [inline debian/control]: Do we need to change `Vcs-Git` and `Vcs-Browser` to this repo?

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Priority: optional
+Maintainer: Boyuan Yang <073plan@gmail.com>
+Build-Depends:
+ debhelper (>= 10),
+ cmake,
+ pkg-config,
+ libwebsockets-dev,
+ libjson-c-dev,
+ libssl-dev,
+ xxd,
```
Comment (tsl0922) [inline debian/control]: The `xxd` package only exists in [stretch (testing)](https://packages.debian.org/search?keywords=xxd) right now, this means the new package won't work for `wheezy` or `jessie` ?

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Source: ttyd
+Section: utils
+Priority: optional
+Maintainer: Boyuan Yang <073plan@gmail.com>
+Build-Depends:
+ debhelper (>= 10),
```
Comment (tsl0922) [inline debian/control]: Use `debhelper (>= 9)`

File: debian/control (hunk 2 above)
Comment (tsl0922) [inline debian/control]: Use `vim-common`

**Comment by hosiet** [comment]
Done, I think we're all set. This should work on Ubuntu 16.04+ and Debian Jessie.

**Pull request merged**